M2/Macaulay2/packages/Matroids.m2

```diff
@@ -285,16 +287,10 @@ flats Matroid := List => M -> (
 	M.cache.flats
 )
 
--- sorts L by values of f (note: L should not involve sequences at all, due to deepSplice)
-sort (List, Function) := opts -> (L, f) -> (
-	H := hashTable((a, b) -> (a, b), apply(L, l -> f(l) => l));
-	deepSplice join apply(sort keys H, k -> H#k)
-)
-
 latticeOfFlats = method()
 latticeOfFlats Matroid := Poset => M -> poset(flats M/toList, (a, b) -> isSubset(a, b))
 
-fVector Matroid := HashTable => opts -> M -> hashTable pairs tally(flats M/rank_M)
+fVector Matroid := HashTable => M -> hashTable pairs tally(flats M/rank_M)

```

Comment:
`fVector` was recently changed to take options, so this change will break your code.  Are you up to date with the master branch and running M2 as built from source code?